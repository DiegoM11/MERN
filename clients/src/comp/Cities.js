import React from 'react';
import logo from '../img/MYtineraryLogo.png';
import home from '../img/homeIcon.png';

class Cities extends React.Component{
  render() {
    return (   
      <div>
        <div>
        <img src={logo} className="App-logo" alt="logo"/>  
        </div>
        <h1>Cities</h1>
        <h2>Buenos Aires</h2>
        <h2>Madrid</h2>
        <h2>Nueva York</h2>
        <h2>Milan</h2>
        <nav>
        <a href="/"><img src={home} className="App-home" alt="home"/></a>
        </nav>
      </div>
  );
  }
}


export default Cities;