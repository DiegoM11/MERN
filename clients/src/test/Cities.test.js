import React from 'react';
import {shallow} from 'enzyme';

import Cities from '../comp/Cities';

it('should render correctly with no props', () => {
    const component = shallow(<Cities/>);
    
    expect(component).toMatchSnapshot();
  });
  it('should render banner text correctly with given strings', () => {
    const strings = ['Cities', 'Madrid', 'Nueva York', 'Milan'];
    const component = shallow(<Cities list={strings} />);
    expect(component).toMatchSnapshot();
  });