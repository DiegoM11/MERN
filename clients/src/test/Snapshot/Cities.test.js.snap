exports['should render correctly with no props 1'] = 'ShallowWrapper {}';
exports ['should render banner text correctly with given strings 1'] = 
        'ShallowWrapper {<h1>Cities</h1>
        <h2>Buenos Aires</h2>
        <h2>Madrid</h2>
        <h2>Nueva York</h2>
        <h2>Milan</h2>}';